### PR TITLE
Add prop to initialOpen option

### PR DESCRIPTION
**Pull request by ygorneves10**
#### What is the purpose of this pull request?

Allow to start open the component

#### What problem is this solving?

The customer needs the Mini Cart to start open. I created a PR in the Drawer repository with this prop too.

#### How should this be manually tested?
https://minicartopen--aldi.myvtex.com/

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by ygorneves10** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch
> * [x]  Minor
> * [ ]  Major
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

File: docs/README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ The VTEX Minicart is a block that displays a summary list of all items added by
 | `drawerSlideDirection` | `Enum`    |  Slide direction for the `'drawer'` Minicart opening. (values: `'rightToLeft'` or `'leftToRight'`) | `rightToLeft` |
 | `linkVariationUrl`     | `String`  |  Link associated to the `'link'` Minicart.                                           |        |
 | `maxDrawerWidth`       | `Number`  | Maximum width (in pixels) for the `'sidebar'` Minicart when opened.                                               | `440`         |
+| `initialOpen`          | `Boolean`  | Minicart starts open.                                                                                   | `false`         |
```
Comment (guerreirobeatriz) [inline docs/README.md]:
```suggestion
| `initialOpen`          | `Boolean`  | Whether the Minicart will be displayed already opened (`true`) or not (`false`).            | `false`         |
```

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]
Thank you!

File: manifest.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "vendor": "vtex",
   "name": "minicart",
   "title": "Mini Cart",
-  "version": "2.43.5",
+  "version": "2.44.0",
```
Comment (klzns) [inline manifest.json]:
```suggestion
  "version": "2.43.5",
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [2.44.0] - 2020-04-03
```
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
```

**Review by klzns — CHANGES_REQUESTED** [review]
The bot will automatically change the version

**Review by klzns — CHANGES_REQUESTED** [review]
I rebased your branch with the current master.

I've tried using this feature but it didnt work. Can you provide a workspace with this linked, please? Thanks

**Comment by Klynger** [comment]
Closing this PR due to inactivity

**Pull request closed without merging**